GDAXSharp/Network/HttpRequest/HttpRequestMessageService.cs

```diff
@@ -45,18 +47,18 @@ public HttpRequestMessageService(IClock clock, bool sandBox)
             var timeStamp = clock.GetTime().ToTimeStamp();
             var signedSignature = ComputeSignature(httpMethod, authenticator.UnsignedSignature, timeStamp, requestUri, contentBody);
 
-            AddHeaders(requestMessage, authenticator, signedSignature, timeStamp);
+            AddHeaders(requestMessage, signedSignature, timeStamp);
             return requestMessage;
         }
 
-        private string ComputeSignature(HttpMethod httpMethod, string secret, double timestamp, string requestUri, string contentBody = "")
+        private static string ComputeSignature(HttpMethod httpMethod, string secret, double timestamp, string requestUri, string contentBody = "")
```

Comment:
any reason this was made static?

Reply:
old habits die hard! Because the method no longer uses and instance variables, and it's not a virtual method..... I swapped it over to Static.

Rule CA1822 in FxCop or Code Analysis states:

"After [marking members as static], the compiler will emit non-virtual call sites to these members which will prevent a check at runtime for each call that ensures the current object pointer is non-null. This can result in a measurable performance gain for performance-sensitive code. In some cases, the failure to access the current object instance represents a correctness issue."